### DISCUSSION TITLE
Amazon Fire TV Cube 2nd Gen - up/down/left/right/center buttons

### DISCUSSION THREAD

**Discussion by shawnhaywood** [PRProd/HA-Firemote#396] [Q&A] [answered]
Hello,  I'm sure this problem is my inexperience.  I added Firemote to HA using the android bridge. Everything works, except the remote direction wheel buttons and select.  However, if I click a corresponding button on the physical remote, the HA Firemote direction buttons start working fine until I turn off the TV/stick.  Any ideas? The app launch buttons and other buttons, like Home, work all the time.

**Comment by PRProd** [comment, marked as answer]
Greetings @shawnhaywood!

What you are describing is a known issue with the Amazon Fire line of products.  This issue is mentioned in the [FAQ](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#faq) (frequently asked questions) section in the Firemote [readme](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file) labelled [Why don't any of the buttons on the Firemote work at all?](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#why-dont-any-of-the-buttons-on-the-firemote-work-at-all). 

At this time, unfortunately there is not a proper fix for this.  You can find [more details about that struggle here](https://github.com/PRProd/HA-Firemote/discussions/115#discussioncomment-5970362).

There is one way to work around this, but using this solution does introduce another behavior that you might not like.  If you switch your Compatibility Mode to  **Strong (Slower)** like this:

![image](https://github.com/PRProd/HA-Firemote/assets/25486599/9d6395ef-a113-4e7c-8b36-96c165c7a5e8)

The disconnection problems you have will disappear, but at a cost.  The navigation buttons will have quite a lag between the time you press the button on your Firemote, and when your device responds.

The best advice I can provide is to either keep your device powered on, keep your physical remote handy for the "press one button" solution, or switch to the **Strong (Slower)** Compatibility Mode.

**Reply by shawnhaywood** [reply]
Thanks so much for the thorough explanation. Sorry I missed the FAQ. Think for now I'll keep the physical remote handy for one press of the button :-)

**Reply by PRProd** [reply]
You're very welcome @shawnhaywood!  That's the solution I prefer to use also.
